README.md

```diff
@@ -32,6 +32,8 @@ These instructions should get you set up ready to work on New Expensify 🙌
 1. Install `node` & `npm`: `brew install node`
 2. Install `watchman`: `brew install watchman`
 3. Install dependencies: `npm install`
+4. Create a `.env` file, use [.env.staging](.env.staging) as a source.
```

Comment:
Can you just not create an `.env` file and use the defaults? Ideally that is what we want for contributors.

Reply:
If it works without that'll be great. 

I don't know why but I had to create `.env` when I started. Maybe the process and/or the proxy handling changed since then so now we don't have to explicitly enable it.

There's also the problem that some External contributors are creating an `.env` file for one reason or another, so we should relay that they should at least use the `staging` template and to warn them about `USE_WEB_PROXY`

Reply:
> Maybe the process and/or the proxy handling changed since then so now we don't have to explicitly enable it.

That's correct, I added logic to make this possible and it's the goal to not have external contributors to create an `.env` file. 

> to warn them about `USE_WEB_PROXY`

I think it's totally fine to add documentation on each environment variable, but it looks like this PR only covers about half of them.

Reply:
> I think it's totally fine to add documentation on each environment variable, but it looks like this PR only covers about half of them.

Yes, I just don't know how to document the rest. Not sure about their purpose. 
I've add only the ones that I do know and the metrics related are not mentioned in `.env.staging` and `.env.example`